manifests/04-rbac-rolebinding.yaml

```diff
@@ -25,19 +25,6 @@ subjects:
 - kind: ServiceAccount
   name: console-operator
 ---
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Moved to `manifests/04-rbac-rolebinding-cluster.yaml` (part of original PR).

Reply:
Which, is not here.  Going to close & reopen, the cherry-pick is missing something.